### ISSUE TITLE
Email Message alerts

### ISSUE THREAD

**Issue by benholtz** [ahertel/wholefoods-delivery-slot-finder#21]
Would love to see this send with some SMTP email with a library sending emails through an authenticated SMTP account like gmail !

**Comment by benholtz** [comment]
this looks promising...  https://gist.github.com/joech4n/d59e7bc9f99273fbda31

**Comment by ahertel** [comment]
@benholtz Thanks for raising this. To make sure I understand: your proposed enhancement is to do what I did for the text message notification feature, but sending emails instead of texts?

While I like the idea, I realized in writing this comment that the text feature will be somewhat obsolete once I release the auto-place order feature(see #6) later today or tomorrow. It would see the email feature, if identical to the text feature, would then be obsolete as well. Let me know what you think the use case would be, if any, for email notification once auto-checkout is released. Thanks!

**Comment by benholtz** [comment]
Yes, indeed to reproduce the alerts for text to email (not everyone uses imessage, but everyone uses email).  I would see it as a 2nd option.  The auto place order is a great feature, but due to the issue of things going out of stock, I can imagine some people may still just want a notice in the case there are out of stock things that are critical to their order.

**Comment by benholtz** [comment]
You might also consider instead making some kind of API call like IFTT that could do the other things (like push notices, emails, turn on and off lights in the house, etc).

**Comment by ahertel** [comment]
@benholtz sorry haven't had the chance to reply. I couldn't find out a way to chat/email you directly so just wanted to say here that I appreciate you stepping up to answer people's questions on issue  pages with the resources I've provided!! This tool got a lot more popular that I expected so I've been inundated with messages on reddit/github, while also trying to work on the code. Just today an article written on [CNBC](https://www.cnbc.com/2020/04/08/how-to-get-a-amazon-fresh-whole-foods-delivery-timeslot.html) about this tool, so LinkedIn is blowing up as well.

Hope to get to discuss this email with more with your soon. Thanks for you patience
